### PR TITLE
add pyneval recipe

### PR DESCRIPTION
**Pull request by bennieHan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyneval/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+build:
+  number: 0
+  entry_points:
+    - pyneval=pyneval.cli.pyneval:run
+    - pyneval_tools=pyneval.cli.pyneval_tools:run
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/pyneval/meta.yaml:20]: I believe only `python` and `pip` are required under `host`

File: recipes/pyneval/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 6707520a7f7e449d5503f9c81c280e4caa20ee789c13a1e852c765f70f6788ba
+
+build:
```
Comment (synapticarbors) [inline recipes/pyneval/meta.yaml:12]: I think this package can be `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

If this is the case, you should change `python` to `python >=3.5` or `python =2.7|>=3.5` etc under both the `run` and `host` sections under `requirements`.

File: recipes/pyneval/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
+{% set name = "pyneval" %}
+{% set version = "0.0.0" %}
```
Comment (synapticarbors) [inline recipes/pyneval/meta.yaml:2]: It looks like the latest version is `0.0.2`. You can keep the current version and then put in PR for the update once the feedstock is created

Comment (bennieHan) [reply recipes/pyneval/meta.yaml:2]: Thank you, sir. I'll fix these problems and try it on a private channel first. Thank you for your recommendations.

Comment (synapticarbors) [reply recipes/pyneval/meta.yaml:2]: You're welcome do that, but the changes should be pretty small and you can just commit to the branch associated with this PR. The recipe only needs a few changes and then I'd be happy to merge it.

**Pull request closed without merging**